### DISCUSSION TITLE
Expose Zitadel UI to AWS ALB

### DISCUSSION THREAD

**Discussion by kareembaher** [zitadel/zitadel#8280] [Show and tell]
Hello
i installed Zitadel using helm chart by following this documentation https://github.com/zitadel/zitadel-charts/blob/main/examples/2-postgres-secure/README.md
i made EKS port forward to Zitadel service and i was able to see the UI and brows normally

Now i want to expose Zitadel using Ingress ALB that i created
so i modified the used values file by the official documentation https://raw.githubusercontent.com/zitadel/zitadel-charts/main/examples/2-postgres-secure/zitadel-values.yaml and edited the **ExternalDomain** value to match my ALB DNS
also i can see the changes reflected correctly on the configmap
However the UI is not working when I try to access the it through the ALB DNS and gives me this error
![image](https://github.com/zitadel/zitadel/assets/54943783/64a7d4ea-c812-4c2f-82da-d416ec9a3e5f)

I made a port forward again an here is a screenshot
![image](https://github.com/zitadel/zitadel/assets/54943783/34187d92-c461-454a-8f9c-ba05ceb4d067)

i found this info on the official documentation, but i think this is in case i installed it direcly on a host
Make 127.0.0.1 available at my.domain. For example, this can be achieved with an entry 127.0.0.1 my.domain in the /etc/hosts file.
Open your favorite internet browser at https://my.domain/ui/console/. You can safely proceed, if your browser warns you about the insecure self-signed TLS certificate. This is the IAM admin users login according to your configuration in the [example-zitadel-init-steps.yaml](https://zitadel.com/docs/assets/files/example-zitadel-init-steps-906f0033be72312912bb607b624e2d62.yaml):

can someone please help and tell me what i can do to fix that

**Comment by fforootd** [comment]
Hm can you share your zitadel config?

I think the problem here will be one of these:

- ExternalDomain needs to be set to the ALB domain
- ExternalPort needs to be set to the ALB port (I guess 443)
- ExternalSecure needs to be set to `true` if TLS is used
